lib/src/util/react_wrappers.dart

```diff
@@ -346,10 +373,12 @@ T getDartComponent<T extends react.Component>(/* ReactElement|ReactComponent|Ele
 /// See: <http://facebook.github.io/react/docs/more-about-refs.html#the-ref-callback-attribute>.
 typedef CallbackRef(ref);
 
-/// Returns a function that chains the callback ref of the provided [element] _(if one exists)_
-/// using [newCallbackRef].
+/// Returns a function that chains the callback or [createRef]-based ref of the provided [element]
+/// with [newCallbackRef].
+///
+/// If there is no existing ref, [newCallbackRef] is used.
 ///
-/// > Throws an [ArgumentError] if [ReactElement.ref] is a `String` ref or otherwise not a [CallbackRef].
+/// Throws an [ArgumentError] if [ReactElement.ref] is a String ref.
 ///
 /// TODO: This method makes assumptions about how react-dart does callback refs for dart components, so this method should be moved there (UIP-1118).
```

Comment:
Should we just do this now? https://jira.atl.workiva.net/browse/CPLAT-928 (move it into react-dart?)

Reply:
I started down that path and it seemed a little annoying but doable, but it looks like I also would have to move `cloneElement` over to do the testing, which would be more of a pain.

I'd advocate for doing that later and not increasing the scope of this ticket.

Reply:
LOL k nvm